README.md

```diff
@@ -431,6 +464,20 @@ const options = {
 };
 ```
 
+<a id="custom-highWaterMark"></a>
+
+#### Custom highWaterMark
+
+Stream on Node.js have a smaller internal buffer size (16Kb, aka `highWaterMark`) from client-side browsers (>1Mb, not consistent across browsers). Because of that, when you are writing an isomorphic app and using `res.clone()`, it will hang with large response in Node. 
+
+Since `3.x` you are able to modify the `highWaterMark` option:
```

Comment:
My advise is don't suggest people use `highWaterMark` as the primary solution, and instead tell them the correct way is to resolve both original response and cloned response using `Promise.All` (it's isomorphic and you no longer need to worry about `highWaterMark`), preferably with an example.